src/ocean/core/array/Search.d

```diff
@@ -1013,6 +1013,101 @@ unittest
     test!("==")( countIf( "gbbbi"[], ( char c ) { return c == 'd'; } ), 0 );
 }
 
+/*******************************************************************************
+
+    Checks if array B is a subset of array A. Array A needs to be sorted.
+    Array B doesn't need to be sorted and might have duplicate elements. It
+    checks if array A contains each item of array B by using bcontains function.
+    If T is a class or struct, comparison is performed using T.opCmp().
+    Otherwise, elements of T are compared using ">" and ">=" or, if T is
+    compatible to size_t (which includes ssize_t, the signed version of size_t),
+    by calculating the difference.
+
+    Template params:
+        T = type of array element
+
+    Params:
+        a = array A
+        b = array B
+
+    Returns:
+        false if there is any item of array B, which is not an item of array A
+
+    In:
```

Comment:
There is no actual contract in code, copy-paste?

Reply:
I copy-pasted from it `bsearch` because this function is using `bcontains` which uses `bsearch`. So the same warning goes for this too.

Reply:
I think it would be more maintainable to just say in regular documentation "See `bsearch` for input constraints" or something like that.